HW_4/tast_1.py

```diff
@@ -0,0 +1,11 @@
+# Задача 36: 
+
+def print_operation_table(operation, num_rows=6, num_columns=6):
+    x = [a + 1 for a in range(num_rows)]
+    y = [b + 1 for b in range(num_columns)]
+
+    for a_x in x:
+       
+     print(*(list(map(operation, [a_x] * num_columns, y))), sep='\t')
```

Comment:
Не плохо. Такого использования функции `map` еще не видел. Но немного поправить не помешает
```suggestion
     print(*map(operation, [a_x] * num_columns, y), sep='\t')
```